sherpa-onnx/c-api/c-api.cc

```diff
@@ -473,10 +473,16 @@ SherpaOnnxKeywordSpotter* CreateKeywordSpotter(
     SHERPA_ONNX_LOGE("%s\n", spotter_config.ToString().c_str());
   }
 
+#ifndef SHERPA_ONNX_ENABLE_WASM_KWS
```

Comment:
Could you disable only the check for hotword file?
We can still validate model files and tokens.txt.

Reply:
ok, thank you for your suggestions